spec/syncSpec.js

```diff
@@ -152,15 +139,59 @@ describe('Sync', function() {
     })
     .then(removeCollision)
     .then(listCollisions)
-    .then(function(collisions) {
+    .then(function verifyNoCollisions(collisions) {
       // There should be no collisions left. We deleted the only one.
       expect(collisions).toEqual({});
     })
     .catch(function(err) {
       expect(err).toBeNull();
     });
-  });
+  }, 60000);
+
+  it('should create records created by other clients', function() {
+    const recordToCreate = { test: 'create' };
+
+    return manage()
+    .then(createRecord(recordToCreate))
+    .then(waitForSyncEvent('record_delta_received'))
+    .then(function verifyDeltaStructure(event) {
+      expect(event.uid).not.toBeNull();
+      expect(event.message).toEqual('create');
+      expect(event.dataset_id).toEqual(datasetId);
+      return event;
+    })
+    .then(doRead())
+    .then(function verifyCorrectRecordApplied(record) {
+      expect(record.data).toEqual(recordToCreate);
+    })
+    .catch(function(err) {
+      expect(err).toBeNull();
+    });
+  }, 60000);
```

Comment:
I think a longer timeout are needed for these specific tasks, but 60 seconds is probably too long.

I will remove the one from the collisions test as that's not needed and I'm not sure why I added it!

Maybe reducing the sync frequency for the dataset on the server (currently 10 seconds I think) would remove the need for this. I will have a look and see.